packages/react-native/React/Views/ScrollView/RCTScrollViewManager.m

```diff
@@ -83,7 +83,7 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(scrollEventThrottle, NSTimeInterval)
 RCT_EXPORT_VIEW_PROPERTY(zoomScale, CGFloat)
 RCT_EXPORT_VIEW_PROPERTY(contentInset, UIEdgeInsets)
-RCT_EXPORT_VIEW_PROPERTY(verticalScrollIndicatorInsets, UIEdgeInsets)
```

Comment:
@cipolleschi I don't see ScrollView props have `verticalScrollIndicatorInsets`, actually it's changed from  `scrollIndicatorInsetsprops` to `verticalScrollIndicatorInsets` in https://github.com/facebook/react-native/pull/44820 , but I don't know why..

Reply:
Yeah, we need to keep the `vertical` version around, because the `RCT_EXPORT_VIEW_PROPERTY` macro exposes the `verticalScrollIndicatorInsets` to JS and there might be product code that uses that prop.
If we change it to `scrollIndicatorInsets`, the product code will break.

Reply:
> `RCTScrollView` not implements verticalScrollIndicatorInsets methods.

How did it work before? 😱

Reply:
I couldn't find any relevant context in the pull request at https://github.com/facebook/react-native/pull/44820. It may just be an oversight. :)